### PR TITLE
add prisma schema and add employees service, controller, module to te…

### PR DESCRIPTION
**Pull request by Summer-luna**
1. Added prisma schema
2. Added employees folder that contains employees service, controller, module for testing if adding a new employee to PostgreSQL database works. 
3. employees folder will modify based on future tasks. If not need right now, I can remove it.
4. Tested with Postman, add employee routing, works fine.
5. Didn't push .env file, not sure if need to push.

### CODE REVIEW

File: packages/server/package.json [hunk 1]
```diff
@@ -18,12 +18,20 @@
...
+    "migrate:deploy": "npx prisma migrate deploy",
+    "prisma:generate": "npx prisma generate",
+    "prisma:studio": "npx prisma studio",
+    "prisma:seed": "npx prisma db seed"
   },
   "dependencies": {
     "@nestjs/common": "^9.0.0",
     "@nestjs/core": "^9.0.0",
     "@nestjs/platform-express": "^9.0.0",
+    "@prisma/client": "latest",
```
Comment (gregfrasco) [inline packages/server/package.json:33]: best to pin this to a version, cant guarantee we will be updating to latest if there are breaking changes in a version

File: packages/server/package.json [hunk 2]
```diff
@@ -43,6 +51,7 @@
     "eslint-plugin-prettier": "^4.0.0",
     "jest": "28.1.3",
     "prettier": "^2.3.2",
+    "prisma": "latest",
```
Comment (gregfrasco) [inline packages/server/package.json:54]: best to pin this to a version, cant guarantee we will be updating to latest if there are breaking changes in a version

File: packages/server/src/employees/employees.controller.ts [hunk 3]
```diff
@@ -0,0 +1,26 @@
+import { Controller, Post, Body } from '@nestjs/common';
```
Comment (gregfrasco) [inline packages/server/src/employees/employees.controller.ts:1]: You can probably leave this in now, but we probably wont be using a controller for the time tracker.

File: packages/server/src/employees/employees.controller.spec.ts [hunk 4]
```diff
@@ -0,0 +1,18 @@
+import { Test, TestingModule } from '@nestjs/testing';
```
Comment (gregfrasco) [inline packages/server/src/employees/employees.controller.spec.ts:1]: You can probably leave this in now, but we probably wont be using a controller for the time tracker.

**Review by gregfrasco — COMMENTED** [review]
Overall good work, minor changes

File: packages/server/package.json (hunk 1 above)
Comment (Summer-luna) [reply packages/server/package.json]: modified

**Review by gregfrasco — APPROVED** [review]

**Pull request merged**